### PR TITLE
WIP readme additions

### PR DESCRIPTION
**Pull request by gr2m**
ping @parkerbxyz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ jobs:
           # optional: set permissions (#TBD)
           permissions_contents: write
           # optional: set repositories
-          repositories: gr2m/my-repo1,gr2m/my-repo2
+          account: gr2m
+          repositories: my-repo1,my-repo2
```
Comment (gr2m) [inline README.md:53]: I forgot: a single token cannot have access across accounts.

File: README.md [hunk 2]
```diff
@@ -49,7 +49,8 @@ jobs:
           # optional: set permissions (#TBD)
```
Comment (gr2m) [inline README.md:49]:
```suggestion
          # optional: set permissions (https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28)
```

File: README.md [hunk 3]
```diff
@@ -6,9 +6,9 @@
 
 In order to use this action, you need to
 
-1. [Register new GitHub App](#TBD)
-2. [Store the App's ID in your repository environment variables](#TBD)
-3. [Store the App's private key in your repository secrets](#TBD)
+1. GitHub App Credentials (ID and private key) ([Register new GitHub App](https://docs.github.com/en/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app))
+2. [Store the App's ID in your repository environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows)
+3. [Store the App's private key in your repository secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository)
```
Comment (parkerbxyz) [inline README.md:11]:
```suggestion
1. GitHub App Credentials (ID and private key) ([Register new GitHub App](https://docs.github.com/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app))
2. [Store the App's ID in your repository environment variables](https://docs.github.com/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows)
3. [Store the App's private key in your repository secrets](https://docs.github.com/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository)
```

Remove the language code from the URLs so users are directed to the language set in their browser.

File: README.md [hunk 4]
```diff
@@ -49,7 +49,8 @@ jobs:
           # optional: set permissions (#TBD)
           permissions_contents: write
           # optional: set repositories
-          repositories: gr2m/my-repo1,gr2m/my-repo2
+          account: gr2m
```
Comment (parkerbxyz) [inline README.md:52]:
```suggestion
          owner: gr2m
```

File: README.md [hunk 5]
```diff
@@ -81,9 +82,20 @@ jobs:
 
 ## How it works
 
-TBD
+The action creates an installation access token using [the `POST /app/installations/{installation_id}/access_tokens` endpoint](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app). By default,
```
Comment (parkerbxyz) [inline README.md:85]:
```suggestion
The action creates an installation access token using [the `POST /app/installations/{installation_id}/access_tokens` endpoint](https://docs.github.com/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app). By default,
```

File: README.md (hunk 4 above)
Comment (gr2m) [reply README.md:52]: Yeah I went back and forth

**Pull request merged**